### ISSUE TITLE
Consider implementing `Debug` for `Client`

### ISSUE THREAD

**Issue by cstyles** [configcat/rust-sdk#3]
### Is your feature request related to a problem? Please describe.

When trying to install a global `configcat::Client` inside a `std::sync::OnceLock` like this:

```rust
use std::sync::OnceLock;

static CONFIGCAT: OnceLock<configcat::Client> = OnceLock::new();

fn main() {
    let client = todo!();
    CONFIGCAT.set(client).unwrap();
}
```

I ran into the following issue:

```
error[E0277]: `Client` doesn't implement `Debug`
    --> src/main.rs:7:27
     |
7    |     CONFIGCAT.set(client).unwrap();
     |                           ^^^^^^ `Client` cannot be formatted using `{:?}` because it doesn't implement `Debug`
     |
     = help: the trait `Debug` is not implemented for `Client`
note: required by a bound in `Result::<T, E>::unwrap`
    --> /Users/cstyles/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/result.rs:1073:12
     |
1071 |     pub fn unwrap(self) -> T
     |            ------ required by a bound in this associated function
1072 |     where
1073 |         E: fmt::Debug,
     |            ^^^^^^^^^^ required by this bound in `Result::<T, E>::unwrap`
```

Essentially, [`OnceLock::<T>::set`](https://doc.rust-lang.org/std/sync/struct.OnceLock.html#method.set) returns a `Result<(), T>` and [`Result::<T, E>::unwrap`](https://doc.rust-lang.org/std/result/enum.Result.html#method.unwrap) requires `E: Debug`.

### Describe the solution you'd like

If `configcat::Client` provided an implementation of `Debug` this would compile. Even if it's just `f.write_str("<Client>")`.

### Describe alternatives you've considered

The lack of a `Debug` implementation can be worked around by mapping the error into something else:

```rust
CONFIGCAT
    .set(client)
    .map_err(|_| "error installing Configcat client")
    .unwrap();
```

### Additional context

Thanks for this library! ❤️

**Comment by z4kn4fein** [comment]
Hi @cstyles, thank you for reporting this issue! I've released a new version (`0.1.2`), which now contains a `Debug` implementation for `Client`.

**Comment by cstyles** [comment]
Thanks for the quick response!
